docs/xdr/features/collect/integrations/endpoint/sekoiaio.md

```diff
@@ -441,15 +536,19 @@ Here are the paths where to find the logs:
     /var/log/endpoint-agent/agent.log
     ```
 
+=== "MacOs"
+
+    ```
+    /var/log/endpoint-agent/agent.log
+    ```
 
 !!! note
-	Please always use the latest Sekoia.io agent version. We recommend activating the update option.
+ Please always use the latest Sekoia.io agent version. We recommend activating the update option.
```

Comment:
```suggestion
    Please always use the latest Sekoia.io agent version. We recommend activating the update option.
```

![image](https://github.com/SEKOIA-IO/documentation/assets/679995/0958792b-1649-4c9c-98df-f196ac3548cb)


instead of

![image](https://github.com/SEKOIA-IO/documentation/assets/679995/c06cdb92-7973-4286-993a-376f09763a5b)